NEWS

```diff
@@ -51,6 +51,8 @@ PHP                                                                        NEWS
   . Fixed bug #74404 (Wrong reflection on DateTimeZone::getTransitions).
     (krakjoe)
   . Fixed bug #74080 (add constant for RFC7231 format datetime). (duncan3dc)
+  . Added DateTime::createFromImmutable() method.
+    (bugs dot php dot net at majkl578 dot cz, Rican7)
```

Comment:
There should be an issue for this, and the issue should be mentioned in the NEWS file, just like the others.